docs/mpr/list-of-mpr-helpers.md

```diff
@@ -23,4 +23,5 @@ Outside of that listed above:
 ## Helpers
 | Name  | MPR Package                                             | Homepage                                                                                 | Programming Language | Search | Clone   | Install | Update | APT-Install | APT-Update | Git-Update               | MPR-Dependencies  |
 | :---  | :------------------------------------------------------ | :--------------------------------------------------------------------------------------- | :------------------- | :----- | :----   | :------ | :----- | :---------- | :--------- | :----------------------- | :---------------  |
+| una   | [una-bin](https://{{env.mpr_url}}/packages/una-bin)             | [https://{{env.github_url}}/AFK-OS/una](https://{{env.github_url}}/AFK-OS/una) | Bash               | Yes    | Yes | Yes     | Yes    | Yes          | Yes         | Yes                       | Yes                | 
```

Comment:
Can you space out the `|` characters to align with those for Tap so everything looks neater?

Not that I'm trying to prioritize my helper, but I planned on keeping any helpers listed here in alphabetical order, so una would probably go below Tap here :p

I also remember you capitalizing `una` as `Una` in the makedeb support rooms a few times - I don't care which way it's done, I just wanted to confirm that you wanted it written as `una`.

Reply:
Sure, I've aligned the pipe characters for una with those for Tap. I've also put Tap above una now.

> I also remember you capitalizing `una` as `Una` in the makedeb support rooms a few times - I don't care which way it's done, I just wanted to confirm that you wanted it written as `una`.

I'm alright with both, but prefer `una` above `Una`.